util/stmtsummary/statement_summary.go

```diff
@@ -721,6 +727,12 @@ func (ssElement *stmtSummaryByDigestElement) add(sei *StmtExecInfo, intervalSeco
 		commitDetails.Mu.Unlock()
 	}
 
+	//plan cache
+	if sei.PlanInCache {
+		ssElement.planInCache = true
+		ssElement.planCacheHits += 1
+	}
+
```

Comment:
It seems `planCacheHits` is enough. Why add `planInCache`?

Reply:
@djshow832 If users use `last_plan_in_cache`, there is no way to view historical plan cache hit information. Add `plan_in_cache`, then you can record the last plan cache hit information for each `sql_digest`.